code_annotations/helpers.py

```diff
@@ -113,25 +114,59 @@ def get_annotation_regex(annotation_regexes):
     """
     Return the full regex to search inside comments for configured annotations.
 
+    A successful match against the regex will return two groups of interest: 'token'
+    and 'data'.
+
+    This regular expression supports annotation tokens that span multiple lines. To do
+    so, prefix each line after the first by two leading spaces. E.g:
```

Comment:
I think this should be "...by at least two leading spaces...", right?  Can you update this comment?

Reply:
:ok_hand: